src/cmd/promremotebench/main.go

```diff
@@ -70,13 +71,23 @@ const (
 	maxNumScrapesActive = 4
 )
 
+type targetUrls []string
+
+func (u *targetUrls) String() string {
+	return "a slice of target urls"
+}
+
+func (u *targetUrls) Set(value string) error {
```

Comment:
nit: Would `Append` be better here? `Set` sounds like a full set and replace op.

Reply:
Unfortunately it has to be `Set` to satisfy `flag.Value`

Reply:
Added a comment to that effect 👍